### PR TITLE
Move rest of the kubemark jobs to prow

### PR DESCRIPTION
**Pull request by krzyzacy**
xref https://github.com/kubernetes/kubernetes/issues/49820
the last three kubemark jobs :-)

/assign @BenTheElder @shyamjvs

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -19947,12 +20075,108 @@ periodics:
...
+    - name: ssh
+      secret:
+        secretName: ssh-key-secret
+        defaultMode: 0400
+    - name: docker-graph
+      hostPath:
+        path: /mnt/disks/ssd0/docker-graph
+
+- name: ci-kubernetes-kubemark-gce-scale
+  interval: 12h
```
Comment (krzyzacy) [inline prow/config.yaml]: (the previous cron is invalid in prow (https://crontab.guru/#H_H/12_*_*_*), and from testgrid it's triggered ~twice a day, so I simply put 12h here.)

**Comment by BenTheElder** [comment]
/lgtm
/hold
TODO (follow up): try to reduce the build up of jobs that are one per node...... this is going to become an issue. (we've been discussing various fixes to reduce this)

**Comment by krzyzacy** [comment]
on the other hand:
prow has 156 nodes while pr-jenkins has 100 + ci-jenkins has ~25 heavy & 10 light agents in total, we should probably still be fine.

in the meanwhile I'm moving towards https://github.com/kubernetes/test-infra/pull/5827

**Comment by BenTheElder** [comment]
> on the other hand:
prow has 156 nodes while pr-jenkins has 100 + ci-jenkins has ~25 heavy & 10 light agents in total, we should probably still be fine.

Prow is running more jobs for more repos than when jenkins was running everything AFAICT, but if we can get back to a bazel-build without the cache we will be in a good place :+1:

**Comment by BenTheElder** [comment]
It looks like the canary in https://github.com/kubernetes/test-infra/pull/5827 is very green.

**Comment by BenTheElder** [comment]
Can we move this to bazel-built without using the cache ssd? or better yet pushing the kubemark image with ci/latest for those jobs? ... :thinking: It would be cool if we could get everything to a run_after_success / shared-build world.

**Comment by krzyzacy** [comment]
yeah https://k8s-testgrid.appspot.com/sig-testing-canaries#kubemark-canary-prow is greener than I expected :-) the [up time for kubemark](https://k8s-testgrid.appspot.com/sig-testing-canaries#kubemark-canary-prow&graph-metrics=test-duration-minutes) seems can take longer sometimes. (cc @shyamjvs )

I'll merge this and make another PR for kubemark ci jobs. This contains a PR job which need to mount cache-ssd anyway

/hold cancel

**Comment by BenTheElder** [comment]
> I'll merge this and make another PR for kubemark ci jobs. This contains a PR job which need to mount cache-ssd anyway

Side note: pull-kubernetes-unit doesn't currently and the impact is minimal, many jobs already take an hour plus, caching git makes only a small difference (though I want to solve concurrent git cache, mulling ideas).

**Comment by BenTheElder** [comment]
Generally when a job is unschedulable recently it is only because of `NodeFitsHostPorts` IE the cache. We have plenty of cpu / mem that could be bin-packed more tightly, when we do #5436 we might want to also just not cache when we won't need to. DinD will always be the worst for caching though since everyone's docker images are ridiculously large compared to other things we download.

**Comment by shyamjvs** [comment]
/lgtm
Thanks for the change.
One question - does having a time period of 12h mean every other run of the job is skipped because the job takes > 12h per run? Or does it just start the next one soon after the current one finishes?

**Comment by krzyzacy** [comment]
@shyamjvs unlike cron, it will queue up until the previous run finishes

**Comment by shyamjvs** [comment]
Thanks. Also could you confirm if there are any more kubemark jobs left to
be migrated (pull or CI)? Sorry I'm too lazy to check (as I'm on my mobile).

**Comment by krzyzacy** [comment]
@shyamjvs they should be all in prow now, I'll close https://github.com/kubernetes/kubernetes/issues/49820 and bikeshed & move some to use bazel built image while you are gone :-)

**Comment by shyamjvs** [comment]
Sounds great - thanks!

**Pull request merged**